util/followthemoney_util/csv.py

```diff
@@ -0,0 +1,37 @@
+import os
+
+import click
+
+from followthemoney_util.cli import cli
+from followthemoney_util.util import read_object
+from followthemoney.export.csv import write_entity, write_headers
+
+
+def _get_csv_handler(outdir, schema, handlers):
+    fh = handlers.get(schema.name)
+    if fh is None:
+        name = "{0}.csv".format(schema.plural)
```

Comment:
This is really cool. Do we need to do something with the schema name to make it safe? I don't think so but it's worth thinking twice :)

Reply:
The name sometimes has space in it. Should we slugify it?